app/src/main/java/org/mifos/mobile/ui/fragments/AboutUsFragment.kt

```diff
@@ -0,0 +1,60 @@
+package org.mifos.mobile.ui.fragments
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import butterknife.BindView
+import butterknife.ButterKnife
+import butterknife.OnClick
+import com.google.android.gms.oss.licenses.OssLicensesMenuActivity
+import org.mifos.mobile.BuildConfig
+import org.mifos.mobile.R
+import org.mifos.mobile.ui.activities.PrivacyPolicyActivity
+import org.mifos.mobile.ui.fragments.base.BaseFragment
+import java.util.*
+
+/*
+~This project is licensed under the open source MPL V2.
+~See https://github.com/openMF/self-service-app/blob/master/LICENSE.md
+*/   class AboutUsFragment : BaseFragment() {
+    @kotlin.jvm.JvmField
+    @BindView(R.id.tv_app_version)
+    var tvAppVersion: TextView? = null
```

Comment:
If we are using **Kotlin** then isn't it better to optimize our code a little more and decrease lines of code. Like, here, I think there is no need for those 3 lines of code(23,24,25) and in all other places. We can simply call `tv_app_version` whenever we want using **Kotlin android-synthetics**

Reply:
Sorry for the late reply on this, I just noticed this comment. Firstly, KotlinX android synthetics is now regarded to be not a recommended practice anymore [[1]](https://android-review.googlesource.com/c/platform/frameworks/support/+/882241) [[2]](https://old.reddit.com/r/androiddev/comments/ala9p2/why_kotlinx_synthetic_is_no_longer_a_recommended/efdvpkg/) since it was replaced with `findViewById`. I did not use that either since the app already used ButterKnife for ViewBinding and I had to primarily focus on completely converting the app to Kotlin first. I will discuss and get back on if I should replace ButterKnife. Thank you for the suggestions and do let me know if there are any more optimization suggestions from your side :)